src/features/common/api/api.ts

```diff
@@ -1,17 +1,13 @@
 import axios from 'axios';
+import {ResponseDataType} from "../types/types";
 
 const instance = axios.create({
     baseURL: 'https://reqres.in/api'
 })
 
-export const getAllUsers = async () => {
-    const response = await  fetch('https://reqres.in/api/users');
-
-    return response.json()
-};
-// export const getAllUsers = async () => {
-//     const response = await instance.get<any>('/users');
-//
-//     return await response.data.json()
-// };
-
+export const reactTestProjectAPI = {
```

Comment:
I would probably suggest sticking to the FP style and use raw functions as much as possible. We can just have the `getAllUsers` function as we did before